tests/Console/GeneratorCommandTest.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace Illuminate\Tests\Console;
+
+use Illuminate\Config\Repository;
+use Illuminate\Console\GeneratorCommand;
+use Illuminate\Console\OutputStyle;
+use Illuminate\Console\View\Components\Factory;
+use Illuminate\Contracts\Filesystem\FileNotFoundException;
+use Illuminate\Contracts\Foundation\Application;
+use Illuminate\Filesystem\Filesystem;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Exception\RuntimeException;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Output\NullOutput;
+
+class GeneratorCommandTest extends TestCase
+{
+    /**
+     * @test
+     */
+    public function itShouldThrowIfNameIsNotPassedAsArgument(): void
```

Comment:
Please use the same `test_` syntax as the rest of the test suite.

Reply:
@driesvints updated...